src/stylesheets/main.scss

```diff
@@ -262,7 +262,17 @@ $colour-list-breakpoint: 980px;
   }
 }
 
-pre {
+// Add styling for inline code
+li code,
+td code,
+p code {
+  padding: 0 govuk-spacing(1);
```

Comment:
Just a note to say `highlight.js` have switched from scrolling `<pre>` to scrolling the inner `<code>`